Userland/Applications/Assistant/main.cpp

```diff
@@ -280,6 +280,10 @@ int main(int argc, char** argv)
     text_box.on_escape_pressed = []() {
         GUI::Application::the()->quit();
     };
+    window->on_active_input_change = [](bool is_active_input) {
+        if (!is_active_input)
+            GUI::Application::the()->quit();
+    };
```

Comment:
I think we want to react to the window become *inactive*, not another window becoming the *active input window*.